Formula/kmod.rb

```diff
@@ -0,0 +1,36 @@
+class Kmod < Formula
+  desc "Linux kernel module handling"
+  homepage "https://git.kernel.org/?p=utils/kernel/kmod/kmod.git"
+  url "https://www.kernel.org/pub/linux/utils/kernel/kmod/kmod-29.tar.xz"
+  sha256 "0b80eea7aa184ac6fd20cafa2a1fdf290ffecc70869a797079e2cc5c6225a52a"
+  license all_of: ["LGPL-2.1-or-later", "GPL-2.0-or-later"]
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gcc" => :build
+  depends_on "libtool" => :build
+  depends_on "m4" => :build
+  depends_on "make" => :build
+  depends_on "pkg-config" => :build
```

Comment:
Not sure if `pkg-config` is being used given there don't seem to be any dependencies outside system glibc. 

Maybe if you were enabling extra features like some Linux distros it may be used, e.g.:
  - Debian https://packages.debian.org/sid/kmod (`openssl@1.1`, `xz`)
  - Arch https://archlinux.org/packages/core/x86_64/kmod/ (`openssl@1.1`, `xz`, `zlib`, `zstd`)

Reply:
Thanks, dropped it as well. :+1: